compose/opensearch/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM opensearchproject/opensearch:latest
```

Comment:
**[Note]** Good catch re-adding this. Must've been deleted in a previous PR by accident 👍🏼